apps/L1G/restaurant/restaurant-frontend/src/components/home/MenuCard.tsx

```
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import React, { useState } from 'react';
+import { Sheet, SheetContent, SheetDescription, SheetHeader, SheetTitle, SheetTrigger } from '@/components/ui/sheet';
```

ESLint

[error] 'Sheet' is defined but never used.

(no-unused-vars)